autopts/wid/gap.py

```diff
@@ -1593,6 +1593,62 @@ def hdl_wid_406(_: WIDParams):
     return True
 
 
+def hdl_wid_450(_: WIDParams):
+    # Please perform LE subrate request procedure.
+
+    stack = get_stack()
+    bd_addr = btp.pts_addr_get()
+    bd_addr_type = btp.pts_addr_type_get()
+
+    params = copy.deepcopy(stack.gap.conn_params.data)
+
+    conn_latency = params.conn_latency
+    supervision_timeout = params.supervision_timeout
+
+    # Magic numbers - PTS does not specify values for subrating procedure
+    subrate_min = 10
```

Comment:
this shouldn't be hardcoded in WID,  lets put this in stack.gap.subrate_min since this may be IUT specific